openmdao/api.py

```diff
@@ -64,6 +64,9 @@
     MultiFiSurrogateModel
 
 from openmdao.utils.find_cite import print_citations
+from openmdao.utils.spline_distributions import cell_centered
```

Comment:
Check with jubs, but my interpretation of the Issue is that he wanted the module moved in, not the individual methods. So, you would do `om.spline_distributions.cell_centered` to get one.

Reply:
He's fine with this